packages/cip2/src/types.ts

```diff
@@ -99,8 +99,6 @@ export type ProtocolParametersForInputSelection = Pick<
   'coinsPerUtxoWord' | 'maxTxSize' | 'maxValueSize' | 'minFeeCoefficient' | 'minFeeConstant'
 >;
 
-export type ProtocolParametersRequiredByInputSelection = Required<
-  {
-    [k in keyof ProtocolParametersForInputSelection]: ProtocolParametersForInputSelection[k];
-  }
->;
+export type ProtocolParametersRequiredByInputSelection = Required<{
```

Comment:
> actually yarn lint:fix changed that.

Hmm, so looks like it's not totally enforced in CI then.

Reply:
It's probably because it doesn't fail on warnings. We could set [--max-warnings 0](https://eslint.org/docs/user-guide/command-line-interface#--max-warnings)